cmd/containerd/config_linux.go

```diff
@@ -11,6 +11,6 @@ func defaultConfig() *config {
 			Level:   "info",
 			Address: "/run/containerd/debug.sock",
 		},
-		Snapshotter: "overlay",
+		DefaultSnapshotter: "snapshot-overlay",
```

Comment:
This isn't right. This should actually only be `overlay`. The dash syntax is for the plugin type.